library/build.gradle

```diff
@@ -67,7 +67,7 @@ dependencies {
     implementation 'io.grpc:grpc-protobuf:1.51.0'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4'
     implementation 'org.web3j:crypto:5.0.0'
-    implementation 'org.xmtp.proto:kotlin:3.12.3'
+    api 'org.xmtp.proto:kotlin:3.12.3'
```

Comment:
This exposes the proto library to the example app which was necessary to call `createConversation(peerAddress, invitation) since invitation is a proto class. I think we _could_ get around this by aliasing invitation, but idk if it's necessary! Would love your input.

Reply:
Hmmm I think we should just alias it. We do a lot of aliasing and I wouldn't want to make other consumers have to be confused by using our proto library since a lot of classes are kinda similar.